<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>LoginDialog</class>
    <widget class="QDialog" name="LoginDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>350</width>
                <height>280</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>CodeColab - Login</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLabel" name="titleLabel">
                    <property name="font">
                        <font>
                            <pointsize>14</pointsize>
                            <weight>75</weight>
                            <bold>true</bold>
                        </font>
                    </property>
                    <property name="text">
                        <string>CodeColab Login</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>
            </item>
            <item>
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeType">
                        <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>10</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <layout class="QFormLayout" name="formLayout">
                    <item row="0" column="0">
                        <widget class="QLabel" name="usernameLabel">
                            <property name="text">
                                <string>Username:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QLineEdit" name="usernameEdit"/>
                    </item>
                    <item row="1" column="0">
                        <widget class="QLabel" name="passwordLabel">
                            <property name="text">
                                <string>Password:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QLineEdit" name="passwordEdit">
                            <property name="echoMode">
                                <enum>QLineEdit::Password</enum>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="0">
                        <widget class="QLabel" name="emailLabel">
                            <property name="text">
                                <string>Email:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="1">
                        <widget class="QLineEdit" name="emailEdit"/>
                    </item>
                    <item row="3" column="1">
                        <widget class="QCheckBox" name="guestCheckBox">
                            <property name="text">
                                <string>Login as Guest</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <spacer name="verticalSpacer_2">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeType">
                        <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>10</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QPushButton" name="registerButton">
                            <property name="text">
                                <string>Register</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="loginButton">
                            <property name="text">
                                <string>Login</string>
                            </property>
                            <property name="default">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>